images/archive/exporter.go

```diff
@@ -25,13 +25,14 @@ import (
 	"path"
 	"sort"
 
+	digest "github.com/opencontainers/go-digest"
+	ocispecs "github.com/opencontainers/image-spec/specs-go"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+
 	"github.com/containerd/containerd/content"
 	"github.com/containerd/containerd/errdefs"
 	"github.com/containerd/containerd/images"
 	"github.com/containerd/containerd/platforms"
-	digest "github.com/opencontainers/go-digest"
-	ocispecs "github.com/opencontainers/image-spec/specs-go"
-	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
Seems accidentally modified?

Reply:
My IDE reordered them, but I generally think it's a good idea to separate in-project packages from dependencies.  Happy to revert if you'd prefer that not to be part of this PR though.

Reply:
I'd prefer that reordering to be in another PR and should be applied to all files at once, with a linter

Reply:
Sounds good, I'll take it out of this one.